src/main/java/com/eva/model/applicant/Applicant.java

```diff
@@ -0,0 +1,89 @@
+package com.eva.model.applicant;
+
+import java.util.Optional;
+import java.util.Set;
+
+import com.eva.model.person.Address;
+import com.eva.model.person.Email;
+import com.eva.model.person.Name;
+import com.eva.model.person.Person;
+import com.eva.model.person.Phone;
+import com.eva.model.tag.Tag;
+
+/**
+ * Represents the Applicant to be stored by Eva.
+ */
+public class Applicant extends Person {
+    protected Optional<InterviewDate> interviewDate;
+    protected ApplicationStatus applicationStatus;
+
+    /**
+     * Creates of an applicant object with a fixed interview date.
+     */
+    public Applicant(Name name,
+                     Phone phone,
+                     Email email,
+                     Address address,
+                     Set<Tag> tags,
+                     InterviewDate interviewDate,
+                     ApplicationStatus status) {
+        super(name, phone, email, address, tags);
+        this.interviewDate = Optional.ofNullable(interviewDate);
```

Comment:
Maybe interviewDate shouldnt be in an optional wrapper in the constructor in case it errors? We could include a try catch then or we could simply make a getter method for interviewDate that returns an optional<InterviewDate> and set up the error handling there.